RELEASE-NOTES.txt

```diff
@@ -1,6 +1,7 @@
 4.3
 -----
 - Products: now the Product details can be edited and saved outside Products tab (e.g. from Order details or Top Performers).
+- In Settings > Experimental Features, a Products switch is now available for turning Products M2 features on and off for simple products (default off for beta testing).
```

Comment:
I think that Products M2 is an internal thing, people don't know about it. Maybe we need to specify what a user can do from now enabling the products switch?

Reply:
good point! added more details about M2 features in https://github.com/woocommerce/woocommerce-ios/pull/2275/commits/1d692fa4aed3d22cefc77c4a27bfefee8b0e5abe - lemme know if it can be further improved!